docs/index.rst

```diff
@@ -0,0 +1,33 @@
+WiFi Login Pages
+================
+
```

Comment:
Let's show at least one image here (the first screnshot in the features section).